std/math.d

```diff
@@ -328,6 +335,9 @@ template floatTraits(T)
             enum EXPPOS_SHORT = 0;
             enum SIGNPOS_BYTE = 0;
         }
+
+        import std.typecons : Tuple;
+        alias Integral = Tuple!(ulong, "significand", ushort);
```

Comment:
> This is doable, but it is definitely more work than I'm ready to devote to supporting a format for which I have no simple and straightforward way of testing

Yes, I know.  I was merely questioning the ``significand`` accessor for the purpose of porting to ``BigEndian`` - this is something that I will get around to, but not today.